### PR TITLE
ft: add contrib.PIL.pixelmatch for comparing PIL.Image instances

### LINKED ISSUE #36: support for pixelmatching directly on PIL.Image objects
I can see that this project has no dependencies currently, and this would add Pillow. But chances are people will already be using Pillow to manipulate their image data, so I think the impact would be minimal.

I would be willing to write a PR that implements this.

### PR DESCRIPTION
**Pull request by Mattwmaster58**
#### Added feature: allowing comparing of PIL.Image instances through `contrib.PIL.pixelmatch` (fixes #36)
- added docs
  - including example
- added tests using the same data as other tests

Much cleaner than cramming it all into one function 😉

### CODE REVIEW

File: pixelmatch/contrib/PIL.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+"""Functions to facilitate direct comparison of PIL.Image instances"""
+from typing import Optional, List, Tuple
+
+from PIL.Image import Image # type: ignore
```
Comment (whtsky) [inline pixelmatch/contrib/PIL.py:4]: Have you considered updating `mypy.ini` instead? https://github.com/whtsky/pixelmatch-py/blob/7ed84e4d57fdc70bb24df5a98102b8c009c933bb/mypy.ini

Comment (Mattwmaster58) [reply pixelmatch/contrib/PIL.py:4]: I originally went this route because it's more explicit, but this method is cleaner and it's a small thing anyways.

**Comment by whtsky** [comment]
Merged, thanks!

**Comment by Mattwmaster58** [comment]
No rush on this, but any estimate on a PyPi release @whtsky ?

**Comment by whtsky** [comment]
About tonight ( UTC+8 ).
I was too tired yesterday, sorry

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Mattwmaster58 <notifications@github.com>
发送时间: Monday, May 11, 2020 8:51:32 AM
收件人: whtsky/pixelmatch-py <pixelmatch-py@noreply.github.com>
抄送: Wu Haotian <whtsky@gmail.com>; Mention <mention@noreply.github.com>
主题: Re: [whtsky/pixelmatch-py] ft: add contrib.PIL.pixelmatch for comparing PIL.Image instances (#42)


No rush on this, but any estimate on a PyPi release @whtsky<https://github.com/whtsky> ?

―

**Comment by whtsky** [comment]
https://pypi.org/project/pixelmatch/0.2.1/

**Pull request merged**